packages/token/src/erc20/extensions/erc4626/erc4626.cairo

```diff
@@ -222,6 +222,12 @@ pub mod ERC4626Component {
         /// Hooks into `InternalImpl::_withdraw`.
```

Comment:
We need to both here and in the doc-site a WARNING or CAUTION entry stating that if you need to call untrusted external contracts we recommend adding reentrancy checks.

Reply:
UPDATE: These are already added in https://github.com/OpenZeppelin/cairo-contracts/tree/audit/v2.0.0-alpha.1, so we don't need to add them here.